### DISCUSSION TITLE
unable to deploy

### DISCUSSION THREAD

**Discussion by david-j-m** [PostOwl/postowl#54] [Help]
Hi, trying to deploy PostOwl to server. The server supports Node.js. In svelte.config.js used **adapter-node**,I ran "npm run build" and "npm run preview", the site previewed ok. I uploaded the content of the build folder, the node_modules folder, package.json and package-lock.json. When I enter [my-py.com](url) I receive "**index of /** my-py.com", what am i doing wrong?
Any help much appreciated

**Comment by keybits** [comment]
Hi David, thanks for getting in touch.

You'll need to configure your web server to access the node app via a reverse proxy. This is the case for all Node.js apps, nothing special about PostOwl / SvelteKit here.

I see you're using LiteSpeed as your web server. I'm not familiar with LitesSpeed, but there are docs on how to do that for OpenLiteSpeed here: https://openlitespeed.org/kb/reverse-proxy-basics/

Hope that helps.
